src/views/MenuView.jsx

```diff
@@ -32,7 +32,7 @@ export const MenuView = () => {
 
   const handleMenuClick = (event) => {
     const targetPage = event.key;
-    navigate(`/targetPage`);
+    navigate(`/` + targetPage);
```

Comment:
you can use string template as follows

```

`${targetPage}\`

```

Reply:
cred ca ai pus / gresit acolo, am incercat cu \ dar nu a functionat